src/rcheevos/condition.c

```diff
@@ -174,6 +174,7 @@ rc_condition_t* rc_parse_condition(const char** memaddr, rc_parse_state_t* parse
       case 'b': case 'B': self->type = RC_CONDITION_SUB_SOURCE; can_modify = 1; break;
       case 'c': case 'C': self->type = RC_CONDITION_ADD_HITS; break;
       case 'd': case 'D': self->type = RC_CONDITION_SUB_HITS; break;
+      case 'e': case 'E': self->type = RC_CONDITION_REMEMBER; can_modify = 1; break;
```

Comment:
I generally try to assign letters that at least indirectly abbreviate the behavior. Given the available letters, I probably would have chosen `K` (for keep). Is there some reason you picked `E`?

Reply:
Similarity to add/sub source and add/sub hits in general use case, so I just followed to the next letter. Is K your reference still? Easy enough to switch if so.

Reply:
Updated it to K.